### DISCUSSION TITLE
Specific Questions about parameters?

### DISCUSSION THREAD

**Discussion by syncronox** [merchantprotocol/theadslab#36] [Open Discussion Forum]
What percentage of profiles have these parameters?

Say I want these parameters:
Credit score ranges
Ethnicities
[Num_Credit_Lines_HH]
Income level
What percentage of profiles will have the parameters above? 

Car make-is the company name?
Car model-specific model like Altima?
Car year-year they purchased or it was manufactured?

Are these parameters yes or no answers?
[Owns_Investments]
[Owns_Stocks_And_Bonds]
[Owns_Mutual_Funds]
[Investment_Type]
[Net_Worth_HH]
[Net_Worth_Midpt_HH]
[Travel_Business]  
[Travel_Personal]  

[Occupation_Detail]. How in detail is this? Can you give me examples?
[Occupation_Type] can you give me examples?
[Occupation_Category] can you give me examples?

How is the data shown in these parameters?
[Weight_Lifting]
[Arts_and_Crafts]
[Boating_Sailing]
[Gambling]

So these show the data in total dollars? How recent? What timeframe?
[Recent_Family_Related_Purchases_Total_Dollars]
[Recent_Family_Related_Purchases_Avg_Dollars]
[Recent_Health_Purchases_Total_Dollars]
[Recent_Health_Purchases_Avg_Dollars]
[Recent_Home_Goods_Living_Purchases_Total_Dollars]
[Recent_Home_Goods_Living_Purchases_Avg_Dollars]

These are yes or no values right? How are these calculated? 
[Home_Furnishing]
[Home_Improvement]
[Recent_Health_Purchases_Total_Orders]
[Recent_Health_Purchases_Total_Items]

Are these yes or no values?
[Auto_BodySubType_Cargov]
[Auto_BodySubType_CommercialCutawayChassis]
[Auto_BodySubType_CrewCab]
[Auto_BodySubType_Crossover]
[Auto_BodySubType_ExtendedCab]
[Auto_BodySubType_ExtendedLengthCargo]
[Auto_BodySubType_ExtendedLengthPassenger]
[Auto_BodySubType_Passenger]
[Auto_BodySubType_RegularCab]
[Auto_BodyType_Chassis]
[Auto_BodyType_Convertible]
[Auto_BodyType_Coupe]
[Auto_BodyType_Crossover]
[Auto_BodyType_Fullsize]
[Auto_BodyType_Hatchback]
[Auto_BodyType_Minivan]
[Auto_BodyType_Sedan]

**Comment by jonathonbyrdziak** [comment]
I don't see the benefit or application in answering this
